fetchComponent.py

```diff
@@ -103,16 +142,20 @@ def buildPackageBoard(packageInfo):
     board["shape"] = [shape]
     return board
 
-def easyEdaToKicad(boardJson):
+def easyEdaToKicad(symbolJson, boardJson):
```

Comment:
You changed the prototype and meaning of the function, but you didn't update the docstring.

Also, you made this function "impure" and inconsistent in what it does:
- first, it takes EasyEDA board and **returns** KiCAD board (in memory)
- second, it takes EasyEDA schematics and **writes** KiCAD sch file into the current working directory. I would consider this as an unclear side-effect.

This looks really confusing when invoked: `kicadBoard = easyEdaToKicad(schSymbol, packageBoard)` - the users might think "Ok, so to create a KiCAD board, I also need a schematics?". Also, he has no clue from the invocation about the newly created sch file.

Why not have two functions, `easyEdaBoardToKicad(boardJson)` and `easyEdaSchematicsToKicad(schematicsJson)`?